modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/SiegeCommand.java

```diff
@@ -202,6 +202,7 @@ private String dayToString(int day){
                 break;
 
         }
+        output = I18nSupport.getInternationalisedString(output);
```

Comment:
What is this for?

Reply:
This is for internationalisation of the day of the week. I placed it after the switch because I'm lazy =P

Reply:
> I feel like these changes to how the strings are labeled will only make it harder for people to make new translations.
> 
> For example, instead of having the English translation on one side and the other language on the other they have to decipher cryptic messages like `Assault - Disabled` or `AssaultInfo - Name` to figure out where they are output in the plugin, and what message makes sense there.
> 
> On top of that, I'd like to see things like "ERROR: " get split into a separate prefix like the movecraft command prefix already is, to simplify translations.

I simply copied what I saw as the convention in the localisation file, sorry if I was incorrect in my evaluation. That being said, I will edit the more cryptic messages and remove the command prefixes from things like "AssaultInfo - Name."

I'll also implement the [ERROR:] prefix, going to put it in red but please contact me if you'd prefer different formatting.

Reply:
Oh no, I'm just saying that *that* specific line is wrong - you're attempting to re-internationalize the output string

Reply:
My bad, I made a clerical mistake